### ISSUE TITLE
Bad first line of LaTeX export

### ISSUE THREAD

**Issue by johnblommers** [vkbo/novelWriter#77]
novelWriter export to LaTeX includes this as the first line:

```latex
\documentclass[12pt]{report}\n\usepackage[utf8]{inputenc}\n\begin{document}\n\newpage
```

This is a newline error. Correct:

```latex
\documentclass[12pt]{report}
\usepackage[utf8]{inputenc}
\begin{document}
\newpage
```

**Comment by vkbo** [comment]
Yes. I checked building this, but must have a mistake when I cleaned up the source afterwards. Will fix ASAP.

**Comment by vkbo** [comment]
Yup, trivial fix. Forgot to remove the raw tag when adding the line breaks.

```python
self.outFile.write(r"\documentclass[12pt]{report}\n")
self.outFile.write(r"\usepackage[utf8]{inputenc}\n")
self.outFile.write(r"\begin{document}\n")
```

**Comment by vkbo** [comment]
Fixed in PR #79.
